src/submission/submission-service.ts

```diff
@@ -641,6 +646,72 @@ export namespace operations {
     );
   };
 
+  export async function getAllCommittedClinicalData(programId: string) {
```

Comment:
can you profile this and see how much time it takes from the api for 4k & 8k of donors ?

Reply:
will do. Hmm but this function shouldn't be awaited I will switch it to be async.

Reply:
what do you mean should not be awaited ?

Reply:
right now I await on the getDonors query. Doesn't make sense to block and wait for mongo to return with the data. I think it makes more sense to do a `.then` callback instead

Reply:
not sure I understand, because `await` and `then` are same thing just different syntax to avoid call back hell